config/metac.yaml

```diff
@@ -9,98 +9,109 @@ spec:
-  - apiVersion: v1
-    resource: services
-    updateStrategy:
-      method: InPlace
-    advancedSelector:
-      selectorTerms:
-      - matchReferenceExpressions:
-        - key: metadata.annotations.dao\.mayadata\.io/openebs-uid
-          refKey: metadata.uid # match this ann value against watch UID
-  - apiVersion: rbac.authorization.k8s.io/v1beta1
-    resource: clusterrolebindings
-    updateStrategy:
-      method: InPlace
-    advancedSelector:
-      selectorTerms:
-      - matchExpressions:
-        - key: metadata.name
-          operator: In
-          values: ["openebs-maya-operator", "openebs-cstor-csi-snapshotter-binding", "openebs-cstor-csi-provisioner-binding",
-                   "openebs-cstor-csi-attacher-binding", "openebs-cstor-csi-cluster-registrar-binding",
-                   "openebs-cstor-csi-registrar-binding"]
-  - apiVersion: rbac.authorization.k8s.io/v1beta1
-    resource: clusterroles
-    updateStrategy:
-      method: InPlace
-    advancedSelector:
-      selectorTerms:
-      - matchExpressions:
-        - key: metadata.name
-          operator: In
-          values: ["openebs-maya-operator", "openebs-cstor-csi-snapshotter-role", "openebs-cstor-csi-provisioner-role",
-                   "openebs-cstor-csi-attacher-role", "openebs-cstor-csi-cluster-registrar-role",
-                   "openebs-cstor-csi-registrar-role"]
-  - apiVersion: v1
-    resource: serviceaccounts
-    updateStrategy:
-      method: InPlace
-    advancedSelector:
-      selectorTerms:
-      - matchReferenceExpressions:
-        - key: metadata.annotations.dao\.mayadata\.io/openebs-uid
-          refKey: metadata.uid # match this ann value against watch UID
-  - apiVersion: apiextensions.k8s.io/v1beta1
-    resource: customresourcedefinitions
-    updateStrategy:
-      method: InPlace
-    advancedSelector:
-      selectorTerms:
-      - matchExpressions:
-        - key: metadata.name
-          operator: In
-          values: ["cstorpoolclusters.openebs.io", "csinodeinfos.csi.storage.k8s.io", "csivolumes.openebs.io",
-                    "volumesnapshotclasses.snapshot.storage.k8s.io", "volumesnapshotcontents.snapshot.storage.k8s.io",
-                    "volumesnapshots.snapshot.storage.k8s.io"]
+    - apiVersion: dao.mayadata.io/v1alpha1
+      resource: openebses
+    - apiVersion: apps/v1
+      resource: daemonsets
+      updateStrategy:
+        method: InPlace
+      advancedSelector:
+        selectorTerms:
+          - matchReferenceExpressions:
+              - key: metadata.annotations.dao\.mayadata\.io/openebs-uid
+                refKey: metadata.uid # match this ann value against watch UID
+    - apiVersion: apps/v1
+      resource: deployments
+      updateStrategy:
+        method: InPlace
+      advancedSelector:
+        selectorTerms:
+          - matchReferenceExpressions:
+              - key: metadata.annotations.dao\.mayadata\.io/openebs-uid
```

Comment:
dao.mayadata.io is a generic domain.
it should be `openebs-upgrade.dao.mayadata.io`

Reply:
We are using `dao.mayadata.io` as the domain only.